lib/koans/02_strings.ex

```diff
@@ -38,4 +38,8 @@ defmodule Strings do
   koan "Strings can be louder when necessary" do
     assert String.upcase("listen") == ___
   end
+
+  koan "They also support interpolation" do
```

Comment:
It might be nice to introduce this a little earlier. Perhaps just under "They can be put together". Thoughts? Also w.r.t. wording, it sounds pretty good, but we might emphasize what interpolation means. Maybe something like "Values may be inserted into strings by interpolation". Is that any clearer?